oak_runtime/src/node/wasm/mod.rs

```diff
@@ -448,8 +529,8 @@ impl WasmInterface {
             .set(actual_length_addr, raw_writer)
             .map_err(|err| {
                 error!(
-                    "{}: channel_read(): Unable to write actual length into guest memory: {:?}",
-                    self.pretty_name, err
+                    "{}: {}(): Unable to write actual length into guest memory: {:?}",
```

Comment:
I think this is confusing in terms of debugging. The function failing to write to guest memory is `channel_read_util`, not what is cotained in `function_name`. I think this applies to all the error logging in this function.